code/datums/outfit.dm

```diff
@@ -347,6 +347,35 @@
 	.["implants"] = implants
 	.["accessory"] = accessory
 
+#define OUTFIT_COPY(var) ##var = target.##var
+/// Copy most vars from another outfit to this one
+/datum/outfit/proc/copy_from(datum/outfit/target)
+	OUTFIT_COPY(name)
+	OUTFIT_COPY(uniform)
+	OUTFIT_COPY(suit)
+	OUTFIT_COPY(toggle_helmet)
+	OUTFIT_COPY(back)
+	OUTFIT_COPY(belt)
+	OUTFIT_COPY(gloves)
+	OUTFIT_COPY(shoes)
+	OUTFIT_COPY(head)
+	OUTFIT_COPY(mask)
+	OUTFIT_COPY(neck)
+	OUTFIT_COPY(ears)
+	OUTFIT_COPY(glasses)
+	OUTFIT_COPY(id)
+	OUTFIT_COPY(l_pocket)
+	OUTFIT_COPY(r_pocket)
+	OUTFIT_COPY(suit_store)
+	OUTFIT_COPY(r_hand)
+	OUTFIT_COPY(l_hand)
+	OUTFIT_COPY(internals_slot)
+	OUTFIT_COPY(backpack_contents)
+	OUTFIT_COPY(box)
+	OUTFIT_COPY(implants)
+	OUTFIT_COPY(accessory)
```

Comment:
not really a need to have this define, it doesnt save on much typing and is arguably less clear